scripts/sync_gokb_packages.groovy

```diff
@@ -39,7 +39,7 @@ while ( moredata ) {
         data.TIPPs.TIPP.each { xmltipp ->
 
           // TIPP.
-          def newtipp = directAddFields (xmltipp, ['medium', 'url', 'series', 'subjectArea'], addCoreItems ( xmltipp ))
+          def newtipp = directAddFields (xmltipp, ['medium', 'url', 'series', 'subjectArea', 'status', 'name', 'lastUpdated', 'publisherName', 'dateFirstInPrint', 'dateFirstOnline', 'publicationType', 'firstAuthor', 'volumeNumber', 'editionStatement', 'firstEditor', 'parentPublicationTitleId', 'precedingPublicationTitleId', 'lastChangedExternal', 'importId'], addCoreItems ( xmltipp ))
```

Comment:
`status` & `name` are handled via `addCoreItems`, `lastUpdated` is immutableand can be disregarded.

Reply:
Done.